homeassistant/components/withings/sensor.py

```diff
@@ -787,6 +788,11 @@ def _async_add_workout_entities() -> None:
             _async_add_workout_entities
         )
 
+    if not entities:
+        LOGGER.warning(
+            "No data found for Withings entry %s, so no sensors will be created"
```

Comment:
```suggestion
            "No data found for Withings entry %s, sensors will be added when new data is available"
```

Reply:
@joostlek You are reviewing your own work now? 🤣

Reply:
@edenhaus messaged me privately with a suggestion to use a better warning, so I added it like this to have a place for discussion :)

Reply:
Well it looks good to me 👍